### DISCUSSION TITLE
"Adjust speed" and "Adjust pointer" options are grey'ed out

### DISCUSSION THREAD

**Discussion by eschmalzer** [mgth/LittleBigMouse#283] [Q&A]
I cannot select the Adjust Speed or Adjust Pointer option.  Is there something I'm doing wrong here?

Monitor 1: Viewsonic VG2439, 94 dpi, 23.5", 1080p, 60hz
Monitor 2: 27mu88w, 162 dpi, 27.2", 4k, 60hz

**Comment by mgth** [comment]
It's grayed out when not all monitors are 100% cause system already deal with that.
